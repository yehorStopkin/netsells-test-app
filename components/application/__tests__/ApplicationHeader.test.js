const { shallowMount } = require('@vue/test-utils');
const ApplicationHeader = require('@/components/application/ApplicationHeader').default;

describe('ApplicationHeader', () => {
    test('component default', () => {
        const wrapper = shallowMount(ApplicationHeader);

        expect(wrapper.exists()).toBeTruthy();
        expect(wrapper.element).toMatchSnapshot();
    });
});
