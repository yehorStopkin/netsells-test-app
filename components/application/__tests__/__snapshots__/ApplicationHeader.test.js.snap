// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationHeader component default 1`] = `
<div
  class="application-header"
>
  <div
    class="application-header__text"
  >
    Netsells
  </div>
   
  <div
    class="application-header__text"
  >
    JavaScript Challenge
  </div>
</div>
`;
